styles.css

```diff
@@ -0,0 +1,101 @@
+
+/* CSS stylesheet */
+.mainheader {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.toolbar {
+  position: absolute;
+  width: 1440px;
+  height: 72px;
+  left: 0;
+  top: 0;
+}
+
+.hamburger {
+  position: relative;
+  top: -114px;
+  right: -305px;
+  align-items: baseline;
+}
+
+.contactme {
+  position: relative;
+  width: 117px;
+  height: 14px;
+  margin-left: -13px;
+  color: #7f8cff;
+}

```

Comment:
- [ ] Please keep in mind that you should follow the given design in typography, font size, font weight, colors, margin, and padding. You have done so well so far 👍. However, there is just one small thing left to do 👇 
Kindly fix the gap here
### **Your design**
![Screenshot1](https://user-images.githubusercontent.com/47625407/120923434-cecc7300-c6ce-11eb-99fe-cdefed178cab.png)
### **[Original design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)**
![Screenshot](https://user-images.githubusercontent.com/47625407/120923444-de4bbc00-c6ce-11eb-91de-22503e5ab14d.png)